sw/airborne/arch/chibios/mcu_periph/gpio_arch.h

```diff
@@ -30,10 +30,15 @@
 #ifndef GPIO_ARCH_H
 #define GPIO_ARCH_H
 
-#include "hal.h"
+#include <hal.h>
```

Comment:
it is not a system library but still an external library and the `<>` format is used in (most) case for ChibiOS and libopencm3 headers